sqlanalytics/resource_sql_global_config.go

```diff
@@ -0,0 +1,123 @@
+package sqlanalytics
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type ConfPair struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// GlobalConfig ...
+type GlobalConfig struct {
+	SecurityPolicy          string            `json:"security_policy,omitempty" tf:"default:DATA_ACCESS_CONTROL"`
+	DataAccessConfig        map[string]string `json:"data_access_config,omitempty"`
+	InstanceProfileARN      string            `json:"instance_profile_arn,omitempty"`
+	EnableServerlessCompute bool              `json:"enable_serverless_compute,omitempty" tf:"default:false"`
```

Comment:
aren't all bools false by default? why do we need `tf:"default:false"`

Reply:
Without it, there is a warning when executing:

```
    The following problems may be the cause of any confusing errors from downstream operations:
      - .enable_serverless_compute: was null, but now cty.False
```

so I decided to remove it